imdbsol.txt

```diff
@@ -1,7 +1,22 @@
 #!/bin/bash
 
+url1="http://www.omdbapi.com/?t="
+url2="&y=&plot=short&r=xml" 
+
+for ((i=1;; i++)); do
+    read name || break;
+	echo $name
+
+url="$url1$name$url2"
+echo $url
+curl $url > imdb.txt
+#curl "http://www.omdbapi.com/?t=Godfather&y=&plot=short&r=xml" > imdb.txt
 t=`cat imdb.txt`
 rating="${t#*imdbRating}"
 rating="${rating%%%%}"
 res=${rating:2:3}	
```

Comment:
This will cause issues, if the length of the `rating` string is changed by the api.